Extended Project 3/lib/logic.rb

```diff
@@ -0,0 +1,58 @@
+# Esta linea está llamando a la clase Socket que tiene Ruby
```

Comment:
I think this comment is not necessary because the code is clear in its syntax.

Reply:
Certainly, but we can also forget socket is included in Ruby's installation and doesn't require a gem. Unlike Node where a ``npm init`` and a ``package.json`` is created, Ruby doesn't have this as a requirement. It is more like good practices to use bundler and create an good README.md